var/spack/repos/builtin/packages/ross/package.py

```diff
@@ -17,6 +17,7 @@ class Ross(CMakePackage):
 
     version("develop", branch="develop")
     version("master", branch="master")
+    version("8.0.0", sha256="5339c9afcf77821fb6ebbcd93b8a1e651c7883e94dff250528815a08f84f1694")
```

Comment:
Note: the cmake_args below is when@:7 only. Is it intentional that you don't set the mpicc and mpicxx values anymore?

Reply:
Actually, based on https://github.com/spack/spack/commit/9b5ce67f1e768298befd2913a6366657a97014cb it seems that the fix is in starting with 7.0.1, so I'm going to resolve this as no action needed.